cli/run.py

```diff
@@ -43,24 +44,42 @@ def kill_process_group_afterwards():
     help="Run an experiment on a scenario",
     context_settings=dict(ignore_unknown_options=True),
 )
+@click.option(
+    "--clean",
+    is_flag=True,
+    default=False,
+    help="Clean previously generated artifacts first",
+)
 @click.option(
     "--envision",
     is_flag=True,
     default=False,
     help="Start up Envision server at the specified port when running an experiment",
 )
+@click.option(
+    "--seed",
+    type=int,
+    default=42,
+    help="Set the base seed of the scenario.",
+)
 @click.option(
     "-p",
     "--envision_port",
     help="Port on which Envision will run.",
     default=None,
 )
 @click.argument(
-    "script_path", type=click.Path(exists=True), metavar="<script>", required=True
+    "example_path", type=click.Path(exists=True), metavar="<example>", required=True
```

Comment:
Perhaps it could still be called as `script`.
```suggestion
    "script_path", type=click.Path(exists=True), metavar="<script>", required=True
```